cryodrgn/commands/analyze_landscape.py

```diff
@@ -371,9 +371,9 @@ def hack_barplot(counts_):
 
     plt.figure()
     counts = Counter(labels)
-    g = hack_barplot([counts[i] for i in range(M)])
+    g = hack_barplot([counts[i] for i in range(M)])  # type: ignore  (bug in Counter type-checking?)
     for i in range(M):
-        g.text(i - 0.1, counts[i] + 2, counts[i])
+        g.text(i - 0.1, counts[i] + 2, counts[i])  # type: ignore  (bug in Counter type-checking?)
```

Comment:
On my local setup, `pyright` reported no errors even if I take out these 2 instances of:
```
# type: ignore  (bug in Counter type-checking?)
```
Just FYI in case these are no longer needed.

Reply:
Do you think we should pin `pyright` in our `dev` dependencies to whatever happens to be the currently green version in the CI? It might get frustrating if one can't debug these locally when they fail in the CI. We can then occasionally bump up the version at both places.